src/main/java/com/quizapp/enterprise/persistence/GameTracker.java

```diff
@@ -82,7 +87,10 @@ public void updatePlayerRound(String gameCode, String playerName, boolean correc
     private void updateGameState(String gameCode){
 
         Game game = getGameByCode(gameCode);
-        if(game.getPlayers().stream().allMatch(player -> player.getRound().isComplete())){
+        if(game.getPlayers()
+                .stream()
+                .allMatch(player -> player.getRound()
+                        .isComplete())){
```

Comment:
reformatted the streams to use indentation so the code may be cleaner and easier to read.
This recommended by the "Iterators and Streams" section of the java best practices summary referenced below.

source: https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3